torchvision/csrc/io/image/cpu/decode_jpeg.cpp

```diff
@@ -67,6 +67,58 @@ static void torch_jpeg_set_source_mgr(
   src->pub.next_input_byte = src->data;
 }
 
+inline unsigned char clamped_cmyk_rgb_convert(
+    unsigned char k,
+    unsigned char cmy) {
+  // Inspired from Pillow:
+  // https://github.com/python-pillow/Pillow/blob/07623d1a7cc65206a5355fba2ae256550bfcaba6/src/libImaging/Convert.c#L568-L569
+  auto v = k * cmy + 128;
```

Comment:
Q Are we sure the type coming from `auto` isn't going to lead to over/underflows?

Reply:
type of v will be int due to 128. I can make it explicitly as int if it is more clear